src/lib/annotations/__tests__/Annotator-test.js

```diff
@@ -24,7 +28,7 @@ describe('lib/annotations/Annotator', () => {
             isMobile: false,
             options: {},
             previewUI: {
-                getAnnotateButton: () => {}
+                getAnnotateButton: (() => {})
```

Comment:
Nope, just wanted to make it clear it was one unit, will remove for consistency.